package.json

```diff
@@ -34,14 +34,17 @@
   },
   "dependencies": {
     "bcrypt": "^5.1.0",
+    "body-parser": "^1.20.1",
```

Comment:
body-parser is deprecated. Use express.json instead. It is a built-in express function that does the same thing